### DISCUSSION TITLE
can?

### DISCUSSION THREAD

**Discussion by Rubbi12** [GriefPrevention/GriefPrevention#1841] [Q&A]
hello, can you make a grief prevention plugin for the nukkit server.

**Comment by bigpresh** [comment]
Never heard of Nukkit before, but a quick Google suggests it's a Bedrock server software.

GriefPrevention is for Spigot-type Java servers, we can't help you with Bedrock stuff, and I'm not aware of any plans from any of the GP team to release a Bedrock version - that would likely be a mammoth undertaking as, as far as I'm aware, Bedrock doesn't expose any kind of Bukkit/Spigot-compatible API.
